http/src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changesets/20231002_drop_columns_in_nodepool_table.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="leonardo" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet logicalFilePath="leonardo" author="qi" id="qi_drop_columns_in_nodepool_table">
+        <dropColumn tableName="NODEPOOL" columnName="dateAccessed"/>
+        <addColumn tableName="APP">
+            <column name="namespace" type="VARCHAR(254)">
+                <constraints nullable="true"/>
```

Comment:
Is namespace nullable now? It seems like the old `namespaceId` was not nullable.

Reply:
Maybe could make `APP.namespace` not-null after the SQL `UPDATE` statement?

Reply:
oh that makes sense as an interim step while migrating the field